Documentation/ReleaseIntegrity/Index.rst

```diff
@@ -0,0 +1,246 @@
+.. include:: ../Includes.txt
+.. highlight:: bash
```

Comment:
Two highlights in a row don't make sense, the last one will win.
Right now the document always uses `code-block` anyway. Therefore `.. highlight` is not used at all.

Reply:
Alright... thus, I'm removing these `highlight` statements